sbc_email/models/import_letters_history.py

```diff
@@ -151,6 +151,81 @@ def button_import(self):
+                    if smb_conn and smb_conn.connect(
+                            SmbConfig.smb_ip, SmbConfig.smb_port):
+                        imported_letter_path = self.env.ref(
+                            'sbc_email.scan_letter_imported').value
+                        listPaths = smb_conn.listPath(
+                            share_nas,
+                            imported_letter_path)
+
+                        # loop in import folder and find pdf corresponding to
+                        # this web letter and eventual attached image
+                        image_ext = None
+                        file_to_save = False
+                        for sharedFile in listPaths:
+                            ext = sharedFile.filename[-4:]
+                            if part_filename == sharedFile.filename[:-4]:
+                                if ext == '.pdf':
+                                    file_to_save = True
+                                else:
+                                    image_ext = ext
+
+                        # move web letter on 'Done' folder on the NAS
+                        if file_to_save:
+                            filename = part_filename + '.pdf'
+                            file_obj = BytesIO()
+                            smb_conn.retrieveFile(
+                                share_nas, imported_letter_path + filename,
+                                file_obj)
+                            file_obj.seek(0)
+                            self._copy_imported_to_done_letter(filename,
+                                                               file_obj,
+                                                               False)
+                            # delete files corresponding to web letter in
+                            # 'Import'
+                            try:
+                                smb_conn.deleteFiles(share_nas,
+                                                     imported_letter_path +
+                                                     filename)
+                            except Exception as inst:
+                                logger.info('Failed to delete pdf web letter'
+                                            .format(inst))
+
+                            # image is attached to this letter so we remove it
+                            if image_ext:
+                                try:
+                                    smb_conn.deleteFiles(share_nas,
+                                                         imported_letter_path +
+                                                         part_filename +
+                                                         image_ext)
+                                except Exception as inst:
+                                    logger.info('Failed to delete attached\
+                                                 image {}'.format(inst))
+
+            if letter.manual_import:
+                self._manage_all_imported_files()
```

Comment:
No because code before is for import config web letter (test in lines 161, 162)
When it's a manual import, there is no import_config set, so code before is not execute